Makefile

```diff
@@ -94,7 +94,7 @@ bootstrap-dashboard: \
 .PHONY: bootstrap-dashboard
 
 build: ## build services image
-	$(COMPOSE) build
+	$(COMPOSE) build --build-arg DOCKER_UID=1000 --build-arg DOCKER_GID=1000
```

Comment:
I don't think it's required to define those here: build args are already defined (with the same `1000` default uid/gid) in the `docker-compose` file.